### ISSUE TITLE
Learning Pod 10 Major Project: RED

### ISSUE THREAD

**Issue by ragejin** [cmadland/edci338-202201#93]

**Comment by cmadland** [comment]
Hey everyone! I'm not familiar with RED as a social media platform...can you share a link?

**Comment by ragejin** [comment]
Here is a link that we found introducing RED, and it might be helpful. 
https://www.latmultilingual.com/blog/understanding-red-the-chinese-shopping-app/

**Comment by ragejin** [comment]
> Hey everyone! I'm not familiar with RED as a social media platform...can you share a link?
Here is a link that we found introducing RED, and it might be helpful.
https://www.latmultilingual.com/blog/understanding-red-the-chinese-shopping-app/

**Comment by cmadland** [comment]
Hello @ragejin @JIA-9 @Jialin1227 @V00979195, I see that Jingjing has a [fork of the Course repository at this link](https://github.com/V00979195/edci338-202201).

It would be great if you could all contribute your work to that branch, then maybe once a week you can send me a pull request to merge all your group's work at once. Let me know if you need assistance with this.

You can see a recording of the [zoom call where I talked about this yesterday for reference](https://bright.uvic.ca/d2l/le/lessons/152781/topics/1620432)

**Comment by cmadland** [comment]
Also, the file names you used include several spaces, periods, and Chinese characters. These may make it challenging to embed the images in your page. I suggest you rename them to something like 'persona1.png'

**Comment by V00979195** [comment]
> 

How can we rename our images, I just rename it on my laptop and upload again.

**Comment by cmadland** [comment]
Yes, but did you get my message about making these changes on your learning pod's fork?

**Comment by V00979195** [comment]
> 

https://github.com/V00979195/edci338-202201/pull/1 
Yes, I got your message. Do you mean this one that I listed(the link). How can I put my members in my fork?
